Feature/check accessibility 6th milestone

AmineDerbal opened the pull request:
In this milestone 

- we check the accessibility of a website.

1. **page title:** we changed the page's title to a better one. ✔️ 
2. **image text alternatives:** we changed the alternatives text to meaningful ones ✔️ 
3. **text headings :** no issues found 👍 
4. **color contrast :** no issues found 👍 
5. **resize:** we fixed the background image size in the about me section for the big screens ✔️ 
6. **interaction:** no issues the tabs and scroll work perfectly 👍 
7. **moving content:**  no issues found ✔️ 
8. **multimedia:** no issues with the audio for text content works (checked in Microsoft edge) 👍 
9. **the basic structure of the page:** no issues found 👍

AmineDerbal committed:
check and update page title

AmineDerbal committed:
fixed images alt text

Commit:
Fix text break for background image

IngDavidTM commented on index.html:
```diff
@@ -104,10 +93,7 @@ <h1>Multi-Post Stories</h1>
             <h1>Profesional Art Printing Data</h1>
```
- [x] You did an amazing job identifying the headings on your portfolio :+1: but remember to only use one `<h1>` tag for all your project. Kindly check this [page](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) to get more information about headings and hierarchy :100:

IngDavidTM reviewed (changes requested):
## Hi @AmineDerbal and @fahim-86  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media2.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif"/>

## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] While you did a wonderful job creating a page title :+1: kindly don't forget that it should describe the content of the page adequately and briefly because it would improve the accessibility of your web page :100:. For example:

- Poor titles:
  - Microverse project | Your portfolio.
  - Portfolio

- Better page titles:
  - Mark's portfolio
  - Mark full-stack web developer

To learn more about titles you can read this [checklist](https://www.w3.org/WAI/test-evaluate/preliminary/#title).

- [x] Good job adding all the required images to your portfolio :+1:, but as a project requirement you should add an `alt` attribute in all the images, including 'Mail', with appropriate alternative text that sufficiently describes the image. It is useful for interacting or understanding page :100:. [Here](https://www.w3.org/WAI/test-evaluate/preliminary/#images) you can find more information about `alt` text :+1:. Also, remember that `<img>` tag is self-closing :+1: and some of them are using `</img>` in your project 

- [x] Wonderful job adding color to each element inside your portfolio :+1: :100: :1st_place_medal: , but while there are some people who cannot read the text if there is not sufficient contrast between the text and the background, as an accessibility requirement your page should have a minimum contrast ratio of at least 4.5:1 for normal-size text :+1:. You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) to help you to check the color contrast :100:
**Checker result**
![Screenshot from 2023-01-24 04-51-13](https://user-images.githubusercontent.com/105117832/214261082-f807dc1d-02e8-4811-ae22-dca4c605c0f8.png)


- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AmineDerbal committed:
fix title to a better one

AmineDerbal committed:
fix heading

AmineDerbal committed:
use better alternative text for images

AmineDerbal committed:
fix color contrast using WCAG Contrast checker tool

AmineDerbal committed:
change alt text in the about me section

AmineDerbal committed:
fix style errors

AmineDerbal committed:
fix title

AmineDerbal commented:
- changed the title to Amine's portfolio
- changed the contrast using WCAG Contrast checker tool
- made sure to only have one h1 heading
- changed the alt text of images to better ones
- All my img tag were self closing, I have no </img> in my code

AmineDerbal committed:
fix alt text errors

KanzaTahreem reviewed (approved):
# Approved 🥳

Hi team 👋

Excellent job implementing the changes suggested by reviewer. Great work nailing the project 💪

**_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**

<img src="https://media4.giphy.com/media/cFkiFMDg3iFoI/giphy.gif"/>

## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Excellent branch name!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Your PR is descriptive, well done!
-  ✅ Commits are descriptive, good work.
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
